lib/Draggable.es6

```diff
@@ -138,6 +138,13 @@ export default class Draggable extends DraggableCore {
     }
   }
 
+  componentWillReceiveProps(nextProps) {
```

Comment:
@STRML could you please explain how to use DraggableCore, or attach a ref to the Draggable el or call setState on it directly?